packages/compass-databases-navigation/src/icon-button.tsx

```diff
@@ -6,22 +6,31 @@ import {
   spacing,
   css,
   cx,
-  uiColors,
 } from '@mongodb-js/compass-components';
 
 const iconContainer = css({
-  color: uiColors.white,
   display: 'block',
   flex: 'none',
   fontSize: 0,
 });
 
+const icons = {
+  normal: css({
+    color: 'var(--icon-color)',
+  }),
+  hovered: css({
+    color: 'var(--hover-icon-color)',
+  }),
+} as const;
+
+type IconMode = 'normal' | 'hovered';
```

Comment:
Small thing, fine to ignore, since `normal` and `hovered` are defined above, we might be able to do
```suggestion
type IconMode = keyof typeof icons;
```